posthog/batch_exports/service.py

```diff
@@ -438,14 +438,16 @@ def create_batch_export_run(
     return run
 
 
-def update_batch_export_run_status(run_id: UUID, status: str, latest_error: str | None) -> BatchExportRun:
+def update_batch_export_run_status(
+    run_id: UUID, status: str, latest_error: str | None, records_completed: int = 0
+) -> BatchExportRun:
```

Comment:
Could just be named `update_batch_export_run` as it's doing more than setting the status.

Reply:
On a broader note, I think we should get rid of the ORM and move to something like aiosql.